docs/mast/mast_obsquery.rst

```diff
@@ -286,7 +286,7 @@ Using "obs_id" instead of "obsid" from the previous example will result in the f
 
 .. doctest-remote-data::
    >>> obs_ids = obs_table[0:2]['obs_id']
-   >>> data_products_by_id = Observations.get_product_list(obs_ids)
+   >>> data_products_by_id = Observations.get_product_list(obs_ids)  # doctest: +IGNORE_OUTPUT
```

Comment:
An expected error can be explicitly tested, too, but nevertheless, this looks good as is.